cmd/sonobuoy/app/operations/gen_plugin.go

```diff
@@ -17,7 +17,14 @@ limitations under the License.
 package operations
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
```

Comment:
It doesn't really matter security-wise, but since we control both ends of this connection and we know plugins are going to be written in Go, we could safely use ECDSA public keys here, which should be quite a bit faster.

If you're interested I'd look at swapping out `crypto/rsa` for `crypto/ecdsa` with the [`P256`](https://golang.org/pkg/crypto/elliptic/#P256) curve. This is along the lines of Mozilla's ["modern compatibility"](https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility) recommendation.

Reply:
oh sure, can do!